cmd/earthly/main.go

```diff
@@ -810,17 +810,37 @@ func newEarthlyApp(ctx context.Context, console conslogging.ConsoleLogger) *eart
 			Usage:  "Edits your Earthly configuration file",
 			Hidden: true, // Experimental.
 			Action: app.actionConfig,
-			Description: `This command takes a path, and a value and sets it in your configuration file.
+			UsageText: `This command takes a path, and a value and sets it in your configuration file.
```

Comment:
Not a standard usage, but there isn't an `Examples` section. This kept the flow while being slightly wrong? Willing to reverse or hard-code in the examples section if needed. Also I could make the `UsageText` include the current english description,, and the `Description` include the examples.

Sample, as it is now:

```
NAME:
   earthly config - Edits your Earthly configuration file

USAGE:
   This command takes a path, and a value and sets it in your configuration file.

  As the configuration file is YAML, the key must be a valid key within the file. You can specify sub-keys by using "." to separate levels.
  If the sub-key you wish to use has a "." in it, you can quote that subsection, like this: git."github.com".

  Values must be valid YAML, and also be deserializable into the key you wish to assign them to.
  This means you can set higher level objects using a compact style, or single values with simple values.

  Only one key/value can be set per invocation.

  To get help with a specific key, do "config [key] --help". Or, visit https://docs.earthly.dev/earthly-config for more details.

DESCRIPTION:
   Set your cache size:
   
     config global.cache_size_mb 1234
   
   Set additional buildkit args, using a YAML array:
   
     config global.buildkit_additional_args ['userns', '--host']
   
   Set a key containing a period:
       
     config git."example.com".password hunter2
   
   Configure a server called example.com which stored git repos under /var/git/repos/name-of-repo.git and allowed access over ssh using port 2222 with the git username, you can configure earthly to recognize it as example.com/name-of-repo by configuring it with:
   
     config git "{example: {pattern: 'example.com/([^/]+)', substitute: 'ssh://git@example.com:2222/var/git/repos/\$1.git', auth: ssh}}"

OPTIONS:
   --dry-run   Print the changed config file to the consile instead of writing it out (default: false)
   --help, -h  show help (default: false)

```

Reply:
looks awesome!

Reply:
I made your sentence a bulleted list:

```
   Set up a whole custom git repository for a server called example.com, using a single-line YAML literal:
     * which stores git repos under /var/git/repos/name-of-repo.git
     * allows access over ssh
     * using port 2222
     * sets the username to git
     * is recognized to earthly as example.com/name-of-repo
   
     config git "{example: {pattern: 'example.com/([^/]+)', substitute: 'ssh://git@example.com:2222/var/git/repos/\$1.git', auth: ssh}}"
```